lang/en.json

```diff
@@ -366,8 +366,29 @@
   }
 },
 
+"ACTOR.AbandonCreationContent": "Discard creation progress and exit the creator?",
```

Comment:
I think it would be good to put creation-specific strings under an `ACTOR.CREATION` namespace rather than having a ton of keys in the base `ACTOR` namespace.